input/resources/StructureDefinition-profile-patient.json

```diff
@@ -44,6 +44,216 @@
+        "id": "Patient.identifier:MedicareID.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "http://hl7.org/fhir/sid/us-mbi"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId",
+        "path": "Patient.identifier",
+        "sliceName": "ProviderClientId",
+        "short": "ProviderClient identifier for this patient",
+        "definition": "ProviderClient identifier for this patient.",
+        "max": "1"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "MR"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "Medical Record Number"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "PROVIDER SYSTEM URL"
+      },
+      {
+        "id": "Patient.identifier:ClientId",
+        "path": "Patient.identifier",
+        "sliceName": "ClientId",
+        "short": "Client identifier for this patient",
+        "definition": "Client identifier for this patient.",
+        "max": "1"
+      },
+      {
+        "id": "Patient.identifier:ClientId.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:ClientId.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "MR"
+      },
+      {
+        "id": "Patient.identifier:ClientId.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "ClientId"
+      },
+      {
+        "id": "Patient.identifier:ProviderId",
+        "path": "Patient.identifier",
+        "sliceName": "ProviderId",
+        "short": "Provider identifier for this patient",
+        "definition": "Provider identifier for this patient.",
+        "max": "1"
+      },
+      {
+        "id": "Patient.identifier:ProviderId.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:ProviderId.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "MR"
+      },
+      {
+        "id": "Patient.identifier:ProviderId.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "ProviderID"
+      },
+      {
+        "id": "Patient.identifier:DACODSId",

```

Comment:
I don't think there is a need for Patient.identifier:DACODSId in the new system.  Do we have an input field on this from the new UI Client/Admission screen?  The new system will intake DACODS surveys and attach them to our internal Patient.id.  Confirm if we need to input this value.  Delete it if we don't need it to input it.

Reply:
We do not have any input field on DAcodsid so removing it

Reply:
Thank you for checking!